rotkehlchen/accounting/transactions.py

```diff
@@ -75,15 +76,15 @@ def process(
                 return 1
 
         # if there is any module specific accountant functionality call it
-        if event_settings.accountant_cb is not None:
+        if event_settings.accountant_cb is not None and isinstance(event, EvmEvent):
```

Comment:
Isn't `accountant_cb is None == True` for anything else than `EvmEvent`?

Reply:
Yes, but for the typing sake in the body of this `if`, I would keep this check.

Reply:
We should not add runtime if checks to satisfy mypy type checks.